index.html

```diff
@@ -106,6 +106,68 @@ <h3>
       <h2>
         Examples
       </h2>
+      <h3>
+        Loading and parsing
+      </h3>
+      <p>
+        A model element whose source path resolves to a valid resource
+        dispatch a `load` event upon the successful fetching and 
+        presentation of the resource data. If the asset fails to load,
+        for instance due to an invalid URL, a failure to fetch,
+        or a failure in the source format's parsing process, the user agent dispatches an 
+        `error` event.
+      </p>
+      <aside class="example" title="Error handling">
+      <pre class="js">
+      const model = document.querySelector("model");
+      model.addEventListener("error", () => {
+        // Handle error 
+      });
+      </pre>
+      </aside>
+      <h3>
+        Processing model contents
+      </h3>
+      <p data-cite="infra geometry-1" >
+        A 3D asset file contains positional data for the surfaces and objects
```

Comment:
The text in this section feels more like spec text than an example (it's a bit uncommon to talk about how the internals work in an example)... I wonder if we should move this or actually simplify all this into an example of usage?

Reply:
@marcoscaceres I think it's not example content at all, yeah - I'll pull it out of this PR. Is there a good section name / description of this kind of content you'd recommend instead?